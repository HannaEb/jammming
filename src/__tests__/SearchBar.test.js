import SearchBar from '../Components/SearchBar/SearchBar.js';
import renderer from 'react-test-renderer';

describe('SearchBar', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<SearchBar />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});