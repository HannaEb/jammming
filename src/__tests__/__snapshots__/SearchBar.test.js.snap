// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar matches the snapshot 1`] = `
<div
  className="SearchBar"
>
  <input
    onChange={[Function]}
    placeholder="Enter A Song, Album, or Artist"
  />
  <button
    className="SearchButton"
    onClick={[Function]}
  >
    SEARCH
  </button>
</div>
`;
